pipelines/katello_pipeline.yml

```diff
@@ -15,18 +15,10 @@
     - vars/katello_base_staging.yml
     - vars/katello_{{ katello_version }}.yml
   roles:
-    - umask
-    - selinux
-    - etc_hosts
-    - disable_ipv6
-    - koji
-    - foreman_server_repositories
-    - role: foreman_client_repositories
-      when: katello_version == "nightly" or katello_version is version('3.9', '>=')
-    - disable_firewall
-    - haveged
-    - update_os_packages
-    - foreman_installer
+    - role: katello
```

Comment:
the `katello` (or rather `foreman`) role does not include `disable_firewall`, `disable_ipv6` and `update_os_packages` (and they shouldn't). but we should keep them here, imho.

they also do not include `foreman_client_repostitories`. not sure about that.